app_flutter/test/utils/golden.dart

```diff
@@ -2,11 +2,16 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'dart:async';
 import 'dart:io';
+import 'dart:typed_data';
 
+import 'package:flutter/foundation.dart';
 import 'package:flutter_test/flutter_test.dart';
 import 'package:path/path.dart' as path;
 
+const double _kGoldenDiffTolerance = 0.005;
```

Comment:
@CaseyHillers I am not sure what should be set here.

Reply:
LGTM